release/release_tests.yaml

```diff
@@ -2182,7 +2182,26 @@
       cluster:
         cluster_compute: compute_configs/compute_gpu_1x1_aws.yaml
 
+    - __suffix__: skip_training.jpeg.local_fs_multi_gpus
+      run:
+        timeout: 1200
+        script: bash image_classification/localfs_image_classification_jpeg/download_input_data_from_s3.sh && RAY_TRAIN_V2_ENABLED=1 python train_benchmark.py --task=localfs_image_classification_jpeg --dataloader_type=ray_data --num_workers=4 --skip_train_step --skip_validation_at_epoch_end
+      cluster:
+        cluster_compute: compute_configs/compute_gpu_1x1_multi_gpus_aws.yaml
+
+    - __suffix__: skip_training.jpeg.local_fs_multi_gpus.tuned
+      run:
+        timeout: 1200
+        script: bash image_classification/localfs_image_classification_jpeg/download_input_data_from_s3.sh && RAY_TRAIN_V2_ENABLED=1 python train_benchmark.py --task=localfs_image_classification_jpeg --dataloader_type=ray_data --num_workers=4 --skip_train_step --skip_validation_at_epoch_end --ray_data_override_num_blocks=20
+      cluster:
+        cluster_compute: compute_configs/compute_gpu_1x1_multi_gpus_aws.yaml
 
+    - __suffix__: skip_training.jpeg.local_fs_multi_gpus.torch_dataloader
+      run:
+        timeout: 1200
+        script: bash image_classification/localfs_image_classification_jpeg/download_input_data_from_s3.sh && RAY_TRAIN_V2_ENABLED=1 python train_benchmark.py --task=localfs_image_classification_jpeg --dataloader_type=torch --num_workers=4 --num_torch_workers=9 --skip_train_step --skip_validation_at_epoch_end
+      cluster:
+        cluster_compute: compute_configs/compute_gpu_1x1_multi_gpus_aws.yaml
```

Comment:
(not blocking this PR)
As we are adding more and more variants, we can run multiple variants in a single release test. So we don't need to repeatedly download data.